### PR TITLE
Tal

### PR DESCRIPTION
**Pull request by zigelnik**
## Proposed Changes and Benefits

I am adding Tal Zigelnik to Before I Die with images and text.

## Screenshots

image

image

### Checklist

- [X] I have thoroughly read and understand [The Contribution Guidelines](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/CONTRIBUTION-GUIDELINES.md).
- [X] The title of this PR follows the `keyword: brief description of change` format, using one of the following keywords:
  - `Feature` - adds new or amends existing user-facing behavior.
  - `Chore` - changes that have no user-facing value, refactors, dependency bumps, etc.
  - `Fix` - bug fixes.
  - `Docs` - fixing typos and adding new content.
- [x] I have listed my proposed changes and their benefits.
- [X] I have linked the corresponding issue in the Issues section.

### CODE REVIEW

**Review by XanderRubio — APPROVED** [review]
Hi @zigelnik,

I wanted to let you know that that preview deployment looks fantastic! Your image and text are both excellent, and all of your file uploads were correct and made a valuable contribution. It's amazing to hear that you plan on going scuba diving with sharks. Have you thought about where in the world you might like to try scuba diving with whale sharks? 

Thank you for taking the time to contribute, Tal! Your work will be merged into the main branch. It was also great to see that the text `Before I Die` was correctly translated to Hebrew where you are based.

File: src/Jsons/Contributors.json
Comment (XanderRubio) [inline src/Jsons/Contributors.json:1]: I added a `.` to your text statement.

**Comment by XanderRubio** [comment]
@zigelnik   Your pull request has been successfully merged!!! Congratulations Bottle with Popping CorkBottle with Popping CorkBottle with Popping CorkDouble Exclamation MarkDouble Exclamation MarkDouble Exclamation Mark

If you haven't already, please go ahead and star Glowing StarGlowing StarGlowing Starthis repo and you are more than welcome to continually contribute to this project by assisting with our [open issues](https://github.com/BeforeIDieCode/BeforeIDieAchievements/issues), or adding your own new open issues, and additionally you can get an idea of ideas we have to continually grow this project further as technology and tech stack continually evolve by looking at the [ROADMAP](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/ROADMAP.md) for the project. Have a great day, and thank you for taking the time to contribute, Tal! You can view your contribution now on the [live link](https://before-i-die-achievements.vercel.app/). Have a great weekend and will be sending you an invite to be a member of the Before I Die Code now.

Xander

**Comment by zigelnik** [comment]
> Hi @zigelnik,
> 
> I wanted to let you know that that preview deployment looks fantastic! Your image and text are both excellent, and all of your file uploads were correct and made a valuable contribution. It's amazing to hear that you plan on going scuba diving with sharks. Have you thought about where in the world you might like to try scuba diving with whale sharks?
> 
> Thank you for taking the time to contribute, Tal! Your work will be merged into the main branch. It was also great to see that the text `Before I Die` was correctly translated to Hebrew where you are based.

Thanks for merging! 
So I already have a diving license and seens several sharks worldwide, but my dream is to see this one beautiful creature.
They do gather occasionally in my country in a city called Eilat which is close to the Red Sea but I want to travel to the far East and see them there as well.
I also surprised to see the Hebrew translation although it needs a Right-to-left fix according to the language properties.

**Comment by XanderRubio** [comment]
> > Hi @zigelnik,
> > I wanted to let you know that that preview deployment looks fantastic! Your image and text are both excellent, and all of your file uploads were correct and made a valuable contribution. It's amazing to hear that you plan on going scuba diving with sharks. Have you thought about where in the world you might like to try scuba diving with whale sharks?
> > Thank you for taking the time to contribute, Tal! Your work will be merged into the main branch. It was also great to see that the text `Before I Die` was correctly translated to Hebrew where you are based.
> 
> Thanks for merging! So I already have a diving license and seens several sharks worldwide, but my dream is to see this one beautiful creature. They do gather occasionally in my country in a city called Eilat which is close to the Red Sea but I want to travel to the far East and see them there as well. I also surprised to see the Hebrew translation although it needs a Right-to-left fix according to the language properties.

I see. So the whale shark is pretty rare to see in only certain parts of the world? I have only swam with small sharks in the French Polynesia islands, but those were harmless, and you could touch them and weren't in danger😅 Very impressive.

Great to hear the comment on the right-to left fix as we had another contributor translate the `Before I Die` to Arabic and mentioned to me how it needed to be from right to left, and he went ahead and fixed this. If you are up for it can you open an issue about fixing the Hebrew translation to be displayed from Right-to-left? I can assist if you need help identifying where in the code to look. Check out pull request #101, where Sherif made the adjustment for translating Arabic. Let me know, and have a nice day!

**Pull request merged**